src/runtime_src/core/pcie/linux/shim.h

```diff
@@ -160,6 +161,16 @@ class shim
   // Exec Buf with hw ctx handle.
   void
   exec_buf(xclBufferHandle boh, xrt_core::hwctx_handle* ctxhdl);
+
+  void
+  get_aie_status_version_info(uint16_t &aie_ver_major, uint16_t &aie_ver_minor);
```

Comment:
Why are we adding functions into the shim? We should support this type of information request through different queries